python/mass_nifti_pic.py

```diff
@@ -0,0 +1,203 @@
+
+    if not smallest_id:
+        message = '\n\tERROR: you must specify a smallest PhysiologyFileID on ' \
+                  'which to run the chunking script using -s or --smallest_id option'
+        print(message)
+        print(usage)
+        sys.exit(lib.exitcode.MISSING_ARG)
+
+    if not largest_id:
+        message = '\n\tERROR: you must specify a largest PhysiologyFileID on ' \
+                  'which to run the chunking script using -l or --largest_id option'
+        print(message)
+        print(usage)
+        sys.exit(lib.exitcode.MISSING_ARG)
+
+    if not smallest_id <= largest_id:
+        message = '\n\tERROR: the value for --smallest_id option is bigger than ' \
+                  'value for --largest_id option'
+        print(message)
+        print(usage)
+        sys.exit(lib.exitcode.INVALID_ARG)
+
+    if os.path.isfile(profile):
+        sys.path.append(os.path.dirname(profile))
+        config_file = __import__(os.path.basename(profile[:-3]))
+    else:
+        message = '\n\tERROR: you must specify a valid profile file.\n' + \
+                  profile + ' does not exist!'
+        print(message)
+        print(usage)
+        sys.exit(lib.exitcode.INVALID_PATH)
+
+    return config_file
+
+
+def make_pic(file_id, config_file, verbose):
+    """
+    Call the function create_imaging_pic of the Imaging class on
+    the FileID provided as argument to this function.
+
+    :param file_id    : FileID of the file for which to create the pic
+     :type file_id    : int
+    :param config_file: path to the config file with database connection information
+     :type config_file: str
+    :param verbose    : flag for more printing if set
+     :type verbose    : bool
+    """
+
+    # database connection
+    db = Database(config_file.mysql, verbose)
+    db.connect()
+
+    # grep config settings from the Config module
+    data_dir = db.get_config('dataDirBasepath')
+
+    # making sure that there is a final / in data_dir
+    data_dir = data_dir if data_dir.endswith('/') else data_dir + "/"
+
+    # load the Imaging object
+    imaging = Imaging(db, verbose)
+
+    # grep the NIfTI file path
+    nii_file_path = imaging.grep_file_path_from_file_id(file_id)
+    if not nii_file_path:
+        print('WARNING: no file in the database with FileID = ' + file_id)
+        return
+    if not re.search('.nii.gz$', nii_file_path):
+        print('WARNING: wrong file type. File ' + nii_file_path + ' is not a .nii.gz file')
+        return
+    if not os.path.exists(data_dir + nii_file_path):
+        print('WARNING: file ' + nii_file_path + ' not found on the filesystem')
+        return
+
+    # grep the time length from the NIfTI file header
+    is_4d_dataset = False
+    length_parameters = imaging.get_nifti_image_length_parameters(data_dir + nii_file_path)
+    if len(length_parameters) == 4:
+        file_parameters['time'] = length_parameters[3]
+        is_4d_dataset = True
+
+    # grep the SessionID of the file
```

Comment:
CandID, not SessionID

Reply:
@xlecours corrected the comment in the code. Thanks!